config/biocypher_config.yaml

```diff
@@ -9,17 +9,18 @@ biocypher:
   head_ontology:
     url: https://github.com/biolink/biolink-model/raw/v3.2.1/biolink-model.owl.ttl
     root_node: entity
+  
 
   tail_ontologies:
 
     cheminf:
       url: http://purl.obolibrary.org/obo/cheminf.owl
-      head_join_node: related to at instance level 
-      tail_join_node: has same connectivity as
+      head_join_node: chemical entity
```

Comment:
Nice work on fixing the ontology issue BTW :)

Reply:
> Nice work on fixing the ontology issue BTW :)

:+1: